packages/web/CLAUDE.md

```
@@ -0,0 +1,217 @@
+### Layer separation
+
+- `app/` — routing and layouts only, no logic — pages import from `views/`
+- `views/` — page-level components, compose from `features/` and `components/`
+- `features/` — all business logic, API calls, and domain-specific hooks
+- `components/` — pure UI components, no business logic or API calls
+- `providers/` — React context providers and TanStack Query setup
+- `lib/` — utilities, API client, env validation
+
+## Project structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)